pkg/systemd/logsJournal.jsx

```diff
@@ -290,6 +291,7 @@ export class JournalBox extends React.Component {
                              actionInProgressText={_("Loading earlier entries")}
                              icon={noLogs ? ExclamationCircleIcon : undefined}
                              isActionInProgress={this.state.loading}
+                             actionVariant="link"
```

Comment:
I find this really weird. Why would this not be a properly looking button any more? This makes the action harder to spot. If that's cleared between Garrett and you, then I'll grudgingly accept. But this makes no sense to me.

Reply:
I think this one can go either way, as a secondary button or link styled button. Given the context of a lot of other text right above it, it might get a bit too lost as a link styled button, so perhaps we should keep it as a secondary button here?